app/src/main/java/com/example/she/ViewContacts.java

```diff
@@ -20,37 +21,50 @@
 
     ListView listView;
     DatabaseHandler db;
-    ArrayList<Pair<String,String>> contactList;
+    ArrayList<Pair<String, String>> contactList;
+    ContactListAdaptor contactListAdaptor;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_view_contacts);
 
+        updateListView();
+    }
+
+    @Override
+    public void editOptionSelected(String phoneNumber) {
+        Toast.makeText(this, "Edit coming soon", Toast.LENGTH_SHORT).show();
+    }
+
+    @Override
+    public void deleteOptionSelected(String phoneNumber) {
+
+        Toast.makeText(this, "Deleted " + phoneNumber, Toast.LENGTH_SHORT).show();
```

Comment:
Good Job,bro!
I think instead of displaying "Deleted "+phoneNumber inside Toast, display "Contact Deleted", will look more professional.
Do confirm with @priyalbhatewara123 before changing.

Reply:
Thank You for your valuable suggestion.

Reply:
Yess, change the toast message to "Contact Deleted".

Reply:
any other changes required Ma'am ?